src/redux/sagas/utils/getEndTimeByEndCondition.ts

```diff
@@ -0,0 +1,51 @@
+import { BigNumber } from 'ethers';
+
+import { StreamingPaymentEndCondition } from '~gql';
+
+// Maximum uint256
+export const TIMESTAMP_IN_FUTURE = BigNumber.from(2).pow(256).sub(1);
```

Comment:
So, although it was me who originally made this number, I've since realised we can just use `constants.MaxUint256` from ethers 😂 

Not really a "change request" but just a tiny detail!